src/controller/src/clusters.rs

```diff
@@ -478,6 +478,7 @@ where
             ClusterRole::User => "user",
         };
         let persist_pubsub_url = self.persist_pubsub_url.clone();
+        let secrets_args = self.secrets_args.to_flags();
```

Comment:
This was intentionally part of the orchestrator/service interface and not the controller/cluster interface. Fine for now to unblock the release, but I’d ultimately like to put the orchestrator back in charge of passing this parameter.

Reply:
sounds good! these args can be passed into the individual orchestrator's probably!